src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return "Color: " + color + ", Number: " + number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
```

Comment:
Don't forget to add access modifiers to your methods. It's a good practice to explicitly define the visibility of your class members.